qa-tests-backend/src/main/groovy/util/SplunkUtil.groovy

```diff
@@ -141,6 +133,26 @@ class SplunkUtil {
         }
     }
 
+    static String createHECToken(int port, String tokenName = "stackrox") {
```

Comment:
would it make sense to pass the `collectorServiceName` here and not hardcode `127.0.0.1`?

Reply:
Given that we do a port forward to localhost, I think using a hardcoded value is fine